MIAJS-949 - Analytics events (#59)
MapsIndoors/mapsindoors.github.io: collections/_web/v3/guides/google_analytics.md

---
title: Google Analytics
parent: guides
layout: tutorial
nav_weight: 1100
published: true
date: 2020-01-15
---

## Overview

> In this, we will take a look at Google Analytics.

Reviewer: ```suggestion
In this, we will take a look at Google Analytics.

```

Google Analytics is a free tool that gives you the opportunity to analyze data to get a deeper understanding of your customers and how they use your app. An example could be to figure out which places is hardest to find on your venue or what users typically search for when looking for a location.

> 

Reviewer: ```suggestion
```

### Get started

> To start seeing stats for your [standard web app](http://clients.mapsindoors.com/):

Reviewer: Link to a demo app?

Author: Link to login page yes :)

1. Create or sign in to your Analytics account:
* Go to [google.com/analytics](https://marketingplatform.google.com/about/analytics/)
* Do one of the following:

>   * To create an account, click Start for free.

Reviewer: ```suggestion
  * To create an account, click "Start for free".
```

>   * To sign in to your account, Click Sign in to Analytics.

Reviewer: ```suggestion
  * To sign in to your account, Click "Sign in to Analytics".
```

1. [Set up a property in your Analytics account](https://support.google.com/analytics/answer/1042508). A [property](https://support.google.com/analytics/answer/2649554?hl=en&ref_topic=1009620) represents your website, and is the collection point in Analytics for the data from your site.
1. Provide your MapsIndoors project manager with your newly created Property ID.

> When done you will within 24 hours start receiving stats and events in your Google Analytics Dashboard.

Reviewer: ```suggestion
After doing this, within 24 hours you will start receiving stats and events in your Google Analytics Dashboard.
```

### Events overview

> When a Property Id is added to your MapsIndoors solution you will see the following events in Google Analytics.

Reviewer: ```suggestion
When a `Property ID` is added to your MapsIndoors solution you will see the following events in Google Analytics.
```
